QuickLayout.Core/ViewModels/TipViewModel.cs

```diff
@@ -37,7 +37,8 @@ public int Generosity
             set
             {
                 _generosity = Limit(value);
-				MvvmCross.Platform.Core.MvxAsyncDispatcher.BeginAsync(() => RaisePropertyChanged(() => Generosity));
+                InvokeOnMainThreadAsync(() => RaisePropertyChanged(() => Generosity));
+				//MvxAsyncDispatcher.BeginAsync();
```

Comment:
Assuming this can be removed?

Reply:
Yes, I've left that just in case there was a better way to call that that I was not aware of.
If you agree I can delete that.

Reply:
Yes, it can be removed. I've left that just in case there was a better way to call `MvxAsyncDispatcher.BeginAsync()` in the current version of MvvmCross that I was not aware of.